### PR TITLE
Add step to enable Ruby 2.7 module on EL8

### PR DESCRIPTION
**Pull request by ehelms**
Cherry-pick into:

* [ ] Foreman 2.3 (Satellite 6.9)
* [ ] Foreman 2.1 (Satellite 6.8)

### CODE REVIEW

File: guides/common/modules/proc_configuring-repositories.adoc [hunk 1]
```diff
@@ -49,6 +49,17 @@ include::proc_configuring-repositories-el.adoc[]
 
 endif::[]
 
+ifdef::foreman-el,katello[]
++
+. Enable Ruby 2.7 module:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-manager} module reset ruby:2.7
```
Comment (evgeni) [inline guides/common/modules/proc_configuring-repositories.adoc:58]: isn't it `dnf module reset ruby` without the version?

**Comment by lzap** [comment]
I am just about to branch off 2.4 series, is this something that should go into 2.4/4.0 or nightly only?

**Comment by evgeni** [comment]
> I am just about to branch off 2.4 series, is this something that should go into 2.4/4.0 or nightly only? I'd appreciate if you could ping me on IRC if I should not branch and wait for this PR to be merged, thanks.

This is 2.5+ material.

**Comment by ehelms** [comment]
@lzap Please do branch, with 2.4 already released, things are going to get confusing what is intended changes for nightly (2.5+) and what are not. For 2.5, will we be branching this repository at the same time as the code repositories?

**Comment by lzap** [comment]
> For 2.5, will we be branching this repository at the same time as the code repositories?

The thing is, RH Docs team is a bit behind, when we branch upstream they usually don't even start working on many feature-bugzillas (bigger changes). For this reason I thought it would be easier to delay branching since this documentation was not official yet.

However Katello 4.0 shakes things a bit now, since this github repo is the official documentation now I think we should be branching on time, yes. I am not particular good at keeping dates tho, so adding this to the list: https://github.com/theforeman/tool_belt/pull/348

**Comment by ehelms** [comment]
@lzap Good to merge this change now that we have branched?

**Pull request merged**